### ISSUE TITLE
purpose of '.add_fn(|x| x * 0.125)' in cifar example

### ISSUE THREAD

**Issue by zeroexcuses** [LaurentMazare/tch-rs#65]
At https://github.com/LaurentMazare/tch-rs/blob/master/examples/cifar/main.rs#L45 we have a line of code where we take a Tensor, then multiply every element by 0.125 .

What is the purpose of this line and why do we do this? (Scaling everything by 1/8th seems pointless). Is this some type of magical constant that causes the algorithm to converge faster? If so, where can I read up more on this?

**Comment by LaurentMazare** [comment]
This model is adapted from [cifar10-fast](https://github.com/davidcpage/cifar10-fast/blob/master/dawn.py) which has the same 0.125. I don't think there is anything deep behind this: you could probably achieve the same effect by tweaking the learning rate and the weight decay accordingly.

**Comment by zeroexcuses** [comment]
I'll look into the Stanford/Dawn paper. Thanks!
